### ISSUE TITLE
Dont calculate average days on recurring tickets

### ISSUE THREAD

**Issue by Nomisdk** [campai-bv/autotask-dashboards-os#52]
Hi

Is there anyway to stop the Resource module from calculating average days open on tickets there is recurring?

The reason why i want this, is be course it makes it hard to see the real average days open, be course if a resource is assigned to a recurring ticket, the day is taking from the day that the ticket was created.

**Comment by coencoppens** [comment]
Hi @Nomisdk,

Unfortunately the API doesn't specify whether a ticket is recurring or not, so I can't make the code filter on that. I've done some digging though, and think I've found a way to figure out whether a ticket is recurring or not.

I agree with you on the fact that they mess up the stats since the created date is "wrong". So let's say I can separate them, would it make sense to be able to leave recurring tickets out of all dashboards completely?

**Comment by Nomisdk** [comment]
Hi again

I found this at the forums for excluding.
"<field>TicketNumber <expression op=\"NotLike\">%.%.%</expression></field>"

https://community.autotask.com/forums/p/28430/63024.aspx

Yeah, that would be a workarround, if that was possible.

**Comment by coencoppens** [comment]
Sweet, that's what I figured to do - all recurring tickets seem to get an additional .xxxx added to their number :-)

But I was thinking: what if you take the created date and the due date, calculate the different and use that to create a virtual created date? That would make the average times make sense again.

**Comment by Nomisdk** [comment]
Hmm cant relay figure that one out, can you give a example? :)

**Comment by coencoppens** [comment]
Say you create a recurring ticket:
- Creation date today (17th februari)
- A due date of 2 days
- Let it recur 1 time in a week

That would make these tickets:
- Create date 17 feb - due date 19 feb
- Create date 24 feb - due date 26 feb

**Comment by Nomisdk** [comment]
Yeah that could work :)

**Comment by Nomisdk** [comment]
Any news at this one?

**Comment by coencoppens** [comment]
@Nomisdk No, sorry. The past few months we've been working on a the v2 of the dashboards; the open source version didn't get any love. I can't promiss this will get fixed soon as the main focus is the v2.
